nominees/berkely-ucb-particle.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Berkely UCB Particle and Temperature Sensor UCB Pats",
+    "description": "UCB Particle and Temperature Sensor is a small, portable data-logging device that uses an optical scattering sensor to measure concentrations of fine particle (~PM2.5) in indoor environments. It saves sampled data until transferred and analyzed by a fit-for-purpose processing software on a computer. An upgraded version, PATS+, which is due to be launched in early 2015, will include other pollutants such as carbon monoxide, carbon dioxide, and black carbon. It logs data to an SD card storage and contains multiple power options. Optional software provides interactive data handling and device management.",
+    "website": "https://berkeleyair.com/",
+    "license": [
```

Comment:
Projects must demonstrate the use of an approved open license. @ijeomaemeruwa this is a mandatory field and is a requirement to become a DPG. Kindly update this. Thanks.